tests/test_graph.py

```diff
@@ -217,16 +258,78 @@ def test_breadth_first_traversal(
     cyclical_graph, create_spaghetti
 ):
     graph1 = create_graph
-    graph2 = create_graph2
-    graph3 = create_graph3
-    c_graph = cyclical_graph
-    spag = create_spaghetti
     assert graph1.breadth_first_traversal(1) == [1, 2, 3, 4, 5]
     assert graph1.breadth_first_traversal(2) == [2, 3, 4, 5]
     assert graph1.breadth_first_traversal(3) == [3, 4, 5]
+
+    graph2 = create_graph2
     assert graph2.breadth_first_traversal(1) == [1, 3, 4, 2, 5]
     assert graph2.breadth_first_traversal(3) == [3, 2, 4, 5]
     assert graph2.breadth_first_traversal(4) == [4]
+
+    c_graph = cyclical_graph
     assert c_graph.breadth_first_traversal(1) == [1, 2, 3, 5, 4, 6, 7]
+
+    graph3 = create_graph3
     assert graph3.breadth_first_traversal(1) == [1, 2, 3, 4, 6, 5]
-    assert spag.breadth_first_traversal(1) == [1, 2, 3, 4, 9, 5, 6, 7, 8]
+
+    spag = create_spaghetti
+    t1_nodes = [1]
+    t2_nodes = [2, 3, 4]
+    t3_nodes = [6, 9, 5, 7, 8]
+    bft_vals = spag.breadth_first_traversal(1)
+    assert bft_vals[0] in t1_nodes
+    for ndx in [1, 2, 3]:
+        assert bft_vals[ndx] in t2_nodes
+    for ndx in [4, 5, 6, 7, 8]:
+        assert bft_vals[ndx] in t3_nodes
+
+
+def test_dijkstra(create_dijk, create_spaghetti):
```

Comment:
While this test is technically correct... it smells bad. Too many asserts for one test.